fetch.bs

```diff
@@ -4426,8 +4426,30 @@ steps:
   <ol>
    <li><p>Let <var>transformStream</var> be a new {{TransformStream}}.
 
-   <li><p>Let <var>identityTransformAlgorithm</var> be an algorithm which, given <var>chunk</var>,
-   <a for=TransformStream lt=enqueue>enqueues</a> <var>chunk</var> in <var>transformStream</var>.
+   <li>
+    <p>Let <var>identityTransformAlgorithm</var> be the following steps given
+    <a>byte sequence</a> <var>chunk</var>:
+
+    <ol>
+     <li><p><a for=TransformStream lt=enqueue>Enqueue</a> <var>chunk</var> in
+     <var>transformStream</var>.
+
+     <li>
+      <p>If <var>response</var>'s <a for=response>URL</a> is null, then:
+
+      <ol>
+       <li><p>Increment <var>response</var>'s <a for=response>body info</a>'s
```

Comment:
Should we also explicitly set response's [cache state](https://fetch.spec.whatwg.org/#concept-response-cache-state) to "local"?

Reply:
Done